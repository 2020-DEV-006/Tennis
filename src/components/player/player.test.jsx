import React from "react";
import Player from "./player";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import renderer from "react-test-renderer";
configure({ adapter: new Adapter() });

describe("Player", () => {
  it("render correctly Player component correctly", () => {
    const player = renderer.create(<Player />).toJSON();
    expect(player).toMatchSnapshot();
  });
});
