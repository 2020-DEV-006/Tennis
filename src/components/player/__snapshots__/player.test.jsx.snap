// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player render correctly Player component correctly 1`] = `
<div>
  <div
    className="playerHeading"
  />
  <div
    className="player"
  />
</div>
`;
